.travis.yml

```diff
@@ -8,6 +8,6 @@ env:
 language: node_js
 
 node_js:
-  - "5"
+  - "6"
```

Comment:
@UltCombo I agree. We can have it be in the allowed failures so that, if a new version of node has incompatibilities, we will be notified but CI will not fail.